config/services.yaml

```diff
@@ -136,6 +136,7 @@ services:
     App\Consumer\CreateHomedirConsumer:
         bind:
             Symfony\Bridge\Monolog\Logger $logger: '@monolog.logger.create_homedir'
+            $fsType: "%env(FILESYSTEM_TYPE)%"
```

Comment:
I like fully spelled out variable names, so you instantly can see what it's for.